core/reactor_common.c

```diff
@@ -1984,13 +1984,21 @@ void initialize(void) {
     start_time = current_tag.time;
 
     #ifdef BIT_32
-    LF_PRINT_DEBUG("Start time: %ldns", start_time);
+        #ifdef MICROSECOND_TIME
+            LF_PRINT_DEBUG("Start time: %ldus", start_time);
+        #else
+            LF_PRINT_DEBUG("Start time: %ldns", start_time);
+        #endif
     #else
-    LF_PRINT_DEBUG("Start time: %lldns", start_time);
+        #ifdef MICROSECOND_TIME
+            LF_PRINT_DEBUG("Start time: %ldus", start_time);
+        #else
+            LF_PRINT_DEBUG("Start time: %ldns", start_time);
+        #endif
     #endif
 
     #ifdef ARDUINO
```

Comment:
Why is this not `#ifdef MICROSECOND_TIME`?

Reply:
+1